pages/common/webtorrent.md

```diff
@@ -0,0 +1,36 @@
+# webtorrent
+
+> The command line interface for WebTorrent.
+> Supports magnets, urls, info hashes and .torrent files.
+
+- Download a torrent:
+
+`webtorrent download "{{torrent_id}}"`
+
+- Stream a torrent to VLC:
```

Comment:
I assume this refers to _VLC Media Player_?

Reply:
Ah, yes. That would make sense to include it 👍